site/app/models/gradeable/Gradeable.php

```diff
@@ -1163,4 +1177,11 @@ public function getAllGradingSections() {
 
         return $sections;
     }
-}
+    //return true if students can currently submit regrades for this assignment, false otherwise
+    public function isRegradeOpen(){
```

Comment:
Can you write a phpdoc comment block for this function (see one of the functions above for the format / example)